fulfillment-sdk/docs/CreatePickWave.md

```diff
@@ -0,0 +1,40 @@
+
+# CreatePickWave
```

Comment:
invenotry relic. I think you need to do a fresh clean generate in a CLEAN directory

Reply:
oh wait we moved pickwaves to fulfillment haha, JK

Reply:
lollll